internal/api/api.go

```diff
@@ -56,20 +56,20 @@ type Parameter struct {
 
 type HTTPVerb string
 
-const (
-	DELETE HTTPVerb = "DELETE"
-	GET    HTTPVerb = "GET"
-	PATCH  HTTPVerb = "PATCH"
-	POST   HTTPVerb = "POST"
-	PUT    HTTPVerb = "PUT"
-)
-
-func ToHTTPVerb(s string) (HTTPVerb, error) {
-	verb := HTTPVerb(strings.ToUpper(s))
+func ToHTTPVerb(s string) (string, error) {
```

Comment:
why do we need this function?

Reply:
To map from string to a http.Method[verb]